.github/ISSUE_TEMPLATE/bug_report_template.yaml

```
@@ -0,0 +1,83 @@
+name: Bug Report
+description: Report a bug encountered while operating Tetragon
+labels: ["kind/community-report", "kind/bug", "needs/triage"]
```

```suggestion
labels: ["kind/bug"]
```
we don't have these labels for now